Top/csound.c

```diff
@@ -334,466 +303,476 @@ static inline const CS_TYPE *StringType(CSOUND *csound) {
 }
 
 static inline const CS_TYPE *AsigType(CSOUND *csound) {
-    return csound->asigType;
+  return csound->asigType;
```

Comment:
Curious, why were these types added here as part of the API? Were these not accessible from csound_standard_types.h?

Reply:
Once I made opcodes as plugins, I got undefined symbols for these. The best solution I could find was to make them
available through the Csound struct.